crates/biome_lsp_converters/src/lib.rs

```diff
@@ -33,72 +32,18 @@ pub enum PositionEncoding {
     Wide(WideEncoding),
 }
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
-pub enum WideEncoding {
```

Comment:
Note how you no longer have

```rust
biome_lsp_converters::WideEncoding
biome_lsp_converters::line_index::LineIndex
```

instead you have

```rust
biome_line_index::WideEncoding
biome_line_index::LineIndex
```

I'm unsure of how strict you all are about breaking changes like this one